bedrock/foundation/templates/foundation/annualreport/2011.html

```diff
@@ -696,164 +696,167 @@ <h2 class="section-title">{{ _('Community') }}</h2>
-                  <h4 class="fn">{{ _('Ioana Chiorean') }}</h4>
-                  <h5>
-                    <span class="role">{{ _('Mozilla Rep, QA Lead') }}</span>
-                    <span class="country-name">{{ _('Romania') }}</span>
-                  </h5>
-                </hgroup>
-                <img class="photo" src="{{ media('img/foundation/annualreport/2011/contributors/ioana-thumb.jpg') }}" alt="">
-                <p class="note">
-                {% trans %}
-                  Ioana has been driving Mozilla hackathons across her native Romania
-                  since 2010.
-                {% endtrans %}
-                </p>
-              </a>
-            </li>
-
-            <li id="story-brian" class="vcard">
-              <a class="contributor contrib-play" href="//videos.cdn.mozilla.net/serv/drafts/Contributorstory8.webm" data-poster="{{ media('img/foundation/annualreport/2011/contributors/brian-poster.jpg') }}">
-                <hgroup>
-                  <h4 class="fn">{{ _('Brian King') }}</h4>
-                  <h5>
-                    <span class="role">{{ _('Mozilla Rep, Add-ons Developer') }}</span>
-                    <span class="country-name">{{ _('Slovenia') }}</span>
-                  </h5>
-                </hgroup>
-                <img class="photo" src="{{ media('img/foundation/annualreport/2011/contributors/brian-thumb.jpg') }}" alt="">
-                <p class="note">
-                {% trans %}
-                  Brian has been contributing to Mozilla for more than a decade and
-                  recently sat on the Mozilla Reps Council.
-                {% endtrans %}
-                </p>
-              </a>
-            </li>
-
-            <li id="story-mohomodou" class="vcard">
-              <a class="contributor contrib-play" href="//videos.cdn.mozilla.net/serv/drafts/Contributorstory9.webm" data-poster="{{ media('img/foundation/annualreport/2011/contributors/mohomodou-poster.jpg') }}">
-                <hgroup>
-                  <h4 class="fn">{{ _('Mohomodou Houssouba') }}</h4>
-                  <h5>
-                    <span class="role">{{ _('Localization Lead') }}</span>
-                    <span class="country-name">{{ _('Mali') }}</span>
-                  </h5>
-                </hgroup>
-                <img class="photo" src="{{ media('img/foundation/annualreport/2011/contributors/mohomodou-thumb.jpg') }}" alt="">
-                <p class="note">
-                {% trans %}
-                  Passionate about reviving Songay, a dialect in his native Mali,
-                  Mohomodou has led localization efforts to get Firefox translated
-                  into Songay.
-                {% endtrans %}
-                </p>
-              </a>
-            </li>
-
-          </ul>
-
+        <div class="jcarousel-container">
+          <button type="button" class="btn-prev">Previous</button>
+          <button type="button" class="btn-next">Next</button>
```

Comment:
These strings should be wrapped for l10n, <code>{{ _('Previous') }}</code> and <code>{{ _('Next') }}</code> (I think we may have these already translated elsewhere).